content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -837,6 +838,7 @@ Each feature gate is designed for enabling/disabling a specific feature:
   when shutting down a node gracefully.
 - `GRPCContainerProbe`: Enables the gRPC probe method for {Liveness,Readiness,Startup}Probe. See [Configure Liveness, Readiness and Startup Probes](/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#define-a-grpc-liveness-probe).
 - `HonorPVReclaimPolicy`: Honor persistent volume reclaim policy when it is `Delete` irrespective of PV-PVC deletion ordering.
+  For details on how this works, check the [blog](/blog/2021/12/15/kubernetes-1-23-prevent-persistentvolume-leaks-when-deleting-out-of-order/).
```

Comment:
We should document this feature properly; if documentation isn't available, we fix that (and if we can't get it fixed, then that's a sign to mark the feature at-risk for the upcoming release).

Reply:
We decided not to add documentation for this feature in 1.23 because this is more like a bug fix. I wonder if it will be confusing to add this in the doc. We can add it if you think it helps.

Reply:
CC @deepakkinni

Reply:
@xing-yang A little clarification. This PR is targeted at dev-1.24, which will become the "main" branch for 1.24 release. We are not adding this to 1.23 anyway.